apps/elewa-group-website/src/app/app.component.html

```diff
@@ -1,2 +1 @@
-<elewa-group-header></elewa-group-header>
-<elewa-group-home-page></elewa-group-home-page>
```

Comment:
This file did not need to change

Add the tag for this component on the home page. Leave the header and homepage here